### PR TITLE
Enforce that Athenz authn filter is executed after CORS filter

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/filter/UserAuthWithAthenzPrincipalFilter.java [hunk 1]
```diff
@@ -29,6 +30,7 @@
  * @author bjorncs
  */
 // TODO Remove this filter once migrated to Okta
+@After({"com.yahoo.vespa.hosted.controller.restapi.filter.AccessControlRequestFilter", "BouncerFilter"})
```
Comment (gjoranv) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/athenz/filter/UserAuthWithAthenzPrincipalFilter.java]: Maybe we should add a 'Provides' annotation to the ACRFilter?

**Review by gjoranv — CHANGES_REQUESTED** [review]
Github requires a comment for some reason.

**Comment by bjorncs** [comment]
@gjoranv PTAL

**Review by gjoranv — APPROVED** [review]
💘

**Pull request merged**